app/frontend/src/components/designer/FormViewerActions.vue

```diff
@@ -0,0 +1,96 @@
+<template>
+  <v-row>
+    <v-col v-if="formId">
+      <router-link :to="{ name: 'UserSubmissions', query: { f: formId } }">
+        <v-btn color="primary" outlined>
+          <span>View All Submissions</span>
+        </v-btn>
+      </router-link>
+    </v-col>
+    <v-col v-if="draftEnabled" class="text-right">
+      <!-- Save a draft -->
+      <span v-if="canSaveDraft" class="ml-2">
+        <v-tooltip bottom>
+          <template #activator="{ on, attrs }">
+            <v-btn
+              @click="$emit('save-draft')"
+              color="primary"
+              icon
+              v-bind="attrs"
+              v-on="on"
+            >
+              <v-icon>save</v-icon>
+            </v-btn>
+          </template>
+          <span>Save as a Draft</span>
+        </v-tooltip>
+      </span>
+
+      <!-- Go to draft edit -->
+      <span v-if="showEditToggle" class="ml-2">
+        <router-link
+          :to="{
+            name: 'UserFormDraftEdit',
+            query: {
+              s: submissionId,
+            },
+          }"
+        >
+          <v-tooltip bottom>
+            <template #activator="{ on, attrs }">
+              <v-btn color="primary" icon v-bind="attrs" v-on="on">
+                <v-icon>mode_edit</v-icon>
+              </v-btn>
+            </template>
+            <span>Edit this Draft</span>
+          </v-tooltip>
+        </router-link>
+      </span>
+    </v-col>
+  </v-row>
+</template>
+
+<script>
+import { FormPermissions } from '@/utils/constants';
+
+export default {
+  name: 'MySubmissionsActions',
+  props: {
+    draftEnabled: {
+      type: Boolean,
+      default: false,
+    },
+    formId: {
+      type: String,
+      default: undefined,
+    },
+    permissions: {
+      type: Array,
+    },
+    readOnly: {
+      type: Boolean,
+      default: false,
+    },
+    submissionId: {
+      type: String,
+      default: undefined,
+    },
+  },
+  data() {
+    return {
+      dialog: false,
```

Comment:
Is the `dialog` element being used at all in the component? If not, we could completely remove the entire data option from this component.

Reply:
Fixed